pages/_app.js

```diff
@@ -103,6 +112,21 @@ export default function App({
     }
   }
 
+  // Add Category
+  async function handleAddCategory(newCategory) {
+    const response = await fetch(CategoryURL, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(newCategory),
+    });
+
+    if (response.ok) {
+      categoryMutate();
+    }
+  }
```

Comment:
Normally you wouldn't pass this function as a prop to the compoents but write it right there in the component itself. with useSWR you can always get the categoryMutate inside any component and trigger the refresh from there. No need for a global function!

Reply:
Oh okay, I think I will have to change that on quite a few api calls in the app then. Not just the category one. I'll save that for my code refactoring :)